app/assets/stylesheets/answers.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Answers controller here.
```

Comment:
But wouldn't that be inline JS if we're using React for said components? cc: @logtheman